docs/components/concepts/backups.md

```diff
@@ -0,0 +1,44 @@
+---
+id: backups
+title: "Backups"
+description: "Learn more about Backups in Camunda Platform 8 SaaS."
+---
+
+You can use the backup feature of Camunda Platform 8 SaaS to regularly back up the state of all of its components (Zeebe, Operate, Tasklist, and Optimize) with _zero downtime_. In case of failures that lead to data loss, you can request to restore the backup.
+
+A backup of Camunda Platform 8 SaaS consists of a backup of Zeebe, Operate, Tasklist, Optimize, and the backup of exported Zeebe records in Elasticsearch. Since the data of these applications are dependent on each other, it is important that the backup is consistent across all components. Therefore, the backup of a Camunda Platform 8 Cluster is taken as a whole.
+
+With backups, you can capture snapshots of your data and applications while they are actively in use, resulting in zero downtime or disruption to your operations. Backups are designed specifically for disaster recovery purposes. Backups should not be used for archival of process data.
+
+## Backup and Cluster Relationship
+
+Backups are created and managed on a per-cluster basis. It is important to be aware that deleting a cluster will also delete all associated backups.
+
+:::caution
+Exercise caution when deleting clusters to avoid unintended loss of backups.
+:::
+
+Your cluster generation needs to be greater or equal to 8.2.4-alpha1 to support backups.
+
+## Rate Limits
+
+The backup feature is available exclusively to [<span class="badge badge--enterprise-only">Camunda Platform Enterprise</span>] users. Each cluster has a limit of three backups. To ensure system stability backup operations are subject to rate limits. Specifically, you can perform a backup operation every 5 hours. However, it is possible to delete a backup and create a new one before the 5-hour rate limit expires.
```

Comment:
Is the rate limiting feature for Enterprise customers only? Or hot backups?

Reply:
In my `review fixes` commit I moved the badge to the top of the page and removed this sentence.

Reply:
Ohh good point. Do you have a recommendation where it should go? Right at the beginning after the explanation what they are?

Backups are enterprise only.

Reply:
I moved it to the top of the page 😄